article/_posts/2020-06-05-Agent57.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: review
+title: "Agent57: Outperforming the human Atari benchmark"
+tags: reinforcement deep-learning RNN
+author: "Antoine Théberge"
+cite:
+    authors: "Adrià Puigdomènech Badia, Bilal Piot, Steven Kapturowski, Pablo Sprechmann, Alex Vitvitskyi, Daniel Guo, Charles Blundell"
+    title:   "Agent57: Outperforming the Atari Human Benchmark"
+    venue:   "Arxiv"
+pdf: ""
+---
+
+
+# Highlights
+
+- Agent57 is able to perform well on all 57 Atari games in the ALE[^1]
+- Q-function is split in two to decompose the contribution of intrinsic and extrinsic rewards
+- A meta-controller modeled as a bandit problem controls the tradeoff between exploration and exploitation
+
+# Introduction
+
+![](/article/images/Agent57/improvements.svg)
```

Comment:
The picture looks huge when displayed. Was this intentional?

Reply:
Kinda, I have scaled it down a bit.